ispdb/pal.kijimadaira.jp.xml

```diff
@@ -4,18 +4,26 @@
     <domain>pal.kijimadaira.jp</domain>
     <displayName>木島平村FTTH</displayName>
     <displayShortName>木島平村FTTH</displayShortName>
+    <incomingServer type="imap">
+      <hostname>imap.pal.kijimadaira.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILLOCALPART%</username>
```

Comment:
thought: There does not seem to be documentation as extensive for this provider as for some other japanese ones, so it's tricky to know if the local part is the right thing to use. However, I wonder if we can make an educated guess here: looking at their DNS (and doing a few DNS and whois lookups), quite a few of them seem to use the same provider CloudMail (https://yournet.freebit.com/service/cloudmail/). So I'm wondering whether it would be more likely that they all use similar settings, in which case it might be safer to assume we should use the whole email address here (as with https://github.com/thunderbird/autoconfig/pull/105, which I've checked also uses CloudMail).